bin/i18n/i18n_script_utils.rb

```diff
@@ -110,7 +110,9 @@ def redact_course_content(source, dest, original, *plugins)
   )
   redacted_data = JSON.parse(stdout)
   KEYS_TO_REDACT.each do |k|
-    source_data[k] = redacted_data[k]
+    if redacted_data[k]
+      source_data[k] = redacted_data[k]
```

Comment:
nit: this can just be `source_data[k] = redacted_data[k] if redacted_data[k]`

Although I'd advocate for it to be `source_data[k] = redacted_data[k] unless redacted_data[k].blank?`

Reply:
Done!